app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+
+  # Returns the full title on a per-page basis.       # Documentation comment
+  def full_title(page_title)                     # Method def, optional arg
+    base_title = I18n.t(".sub_title")  # Variable assignment
+    if page_title.empty?                              # Boolean test
```

Comment:
Dùng toán tử 3 ngôi trường hợp code ngắn thì sẽ gọn hơn nhé
page_title.empty? ? base_title : "#{page_title]  | #{base_title}"